src/Compute/Compute.Autorest/custom/New-AzGalleryApplicationVersion.ps1

```diff
@@ -12,24 +12,22 @@ Create or update a gallery Application Version.
 .Description
 Create or update a gallery Application Version.
 #>
-
-
 function New-AzGalleryApplicationVersion {
```

Comment:
PowerShell has strict format rules on comment-based documents, there should be no more than 2 empty lines between the comment and the function, so I removed them.